pkg/Microsoft.Windows.Compatibility.Shims/Microsoft.Windows.Compatibility.Shims.pkgproj

```diff
@@ -5,41 +5,38 @@
   <PropertyGroup>
     <PackageVersion>$(CompatibilityShimsPackageVersion)</PackageVersion>
     <_ShimsLocationPath>$(BaseIntermediateOutputPath)shims/netcoreapp/facades/</_ShimsLocationPath>
+    <SkipPackageFileCheck>true</SkipPackageFileCheck>
+    <SkipValidatePackage>true</SkipValidatePackage>
   </PropertyGroup>
 
   <ItemGroup>
-    <ShimsToInclude Include="mscorlib.dll" />
-    <ShimsToInclude Include="System.dll" />
-    <ShimsToInclude Include="System.Configuration.dll" />
-    <ShimsToInclude Include="System.Core.dll" />
-    <ShimsToInclude Include="System.Data.dll" />
-    <ShimsToInclude Include="System.Drawing.dll" />
-    <ShimsToInclude Include="System.Net.dll" />
-    <ShimsToInclude Include="System.Security.dll" />
-    <ShimsToInclude Include="System.ServiceModel.Web.dll" />
-    <ShimsToInclude Include="System.ServiceProcess.dll" />
-    <ShimsToInclude Include="System.Transactions.dll" />
-    <ShimsToInclude Include="WindowsBase.dll" />
-  </ItemGroup>
 
-  <Target Name="AddShimsToPackage"
-          BeforeTargets="GetPackageFiles">
+    <File Include="$(PlaceHolderFile)">
+      <TargetPath>/lib/netstandard2.0</TargetPath>
```

Comment:
Do we even want this package to install on netstandard2.0?  /cc @weshaggard

Reply:
Yes I think we do given it is part of the Compat package closure and we wanted that to install for NS2.0

Reply:
> part of the Compat package closure

It shouldn't be for netstandard2.0, if it is then that dependency should be conditioned so that it only applies to netcoreapp2.0.

Reply:
OK if it is correctly conditioned then I agree we could just make this fail to install on netstandard.